scripts/influxdb.service

```diff
@@ -9,9 +9,11 @@ After=network.target
 User=influxdb
 Group=influxdb
 LimitNOFILE=65536
+Environment='STDOUT=/var/log/influxdb/influxd.log'
```

Comment:
This is not what we do by default. By default it goes to `/dev/null.`. So is this what you want?

https://github.com/influxdb/influxdb/blob/master/scripts/init.sh#L59